presto-main/src/main/java/io/prestosql/operator/scalar/VersionFunction.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.operator.scalar;
+
+import com.google.common.collect.ImmutableList;
+import io.airlift.slice.Slice;
+import io.prestosql.annotation.UsedByGeneratedCode;
+import io.prestosql.metadata.BoundVariables;
+import io.prestosql.metadata.FunctionMetadata;
+import io.prestosql.metadata.Metadata;
+import io.prestosql.metadata.Signature;
+import io.prestosql.metadata.SqlScalarFunction;
+
+import java.lang.invoke.MethodHandle;
+
+import static io.airlift.slice.Slices.utf8Slice;
+import static io.prestosql.metadata.FunctionKind.SCALAR;
+import static io.prestosql.spi.type.VarcharType.createVarcharType;
+import static io.prestosql.util.Reflection.methodHandle;
+
+public final class VersionFunction
+        extends SqlScalarFunction
+{
+    private static final MethodHandle METHOD_HANDLE = methodHandle(VersionFunction.class, "getVersion", String.class);
+    private final String nodeVersion;
+
+    public VersionFunction(String nodeVersion)
+    {
+        super(new FunctionMetadata(
+                new Signature(
+                        "version",
+                        ImmutableList.of(),
+                        ImmutableList.of(),
+                        createVarcharType(nodeVersion.length()).getTypeSignature(),
```

Comment:
We should pick a fixed return type for this function. Otherwise, the type will change from version to version (especially, for non-release builds, which tend to have longer version numbers). Let's go with `VARCHAR`, since we can't predict what the maximum length of the version string will be in advance.